mappings/net/minecraft/block/AbstractPressurePlateBlock.mapping

```diff
@@ -1,4 +1,18 @@
 CLASS net/minecraft/class_1445 net/minecraft/block/AbstractPressurePlateBlock
+	FIELD field_12565 BOX Lnet/minecraft/class_231;
+	FIELD field_17735 PRESSED_SHAPE Lnet/minecraft/class_4100;
+	FIELD field_17736 DEFAULT_SHAPE Lnet/minecraft/class_4100;
+	METHOD method_11548 updatePlateState (Lnet/minecraft/class_1150;Lnet/minecraft/class_2552;Lnet/minecraft/class_2232;I)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 output
+	METHOD method_11549 playPressSound (Lnet/minecraft/class_3598;Lnet/minecraft/class_2552;)V
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_11550 playDepressSound (Lnet/minecraft/class_3598;Lnet/minecraft/class_2552;)V
```

Comment:
unpress

Reply:
blame modmuss, like 99% of the mappings were copy pasted from upstream

Reply:
👍 fine then